app/controllers/admin_users_account_suspensions_controller.rb

```diff
@@ -24,11 +24,7 @@ def set_suspended_user
   end
```

Comment:
Do we need this controller/routes at all now. Hasn't it been replaced by the 3 new ones?

Reply:
I think this controller is still needed for the "Ban for Spamming" button before you get to the edit page.